egs/timit/ASR/prepare.sh

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env bash
+
+set -eou pipefail
+
+num_phones=39
+# Here we use num_phones=39 for modeling
+
+nj=15
+stage=-1
+stop_stage=100
+
+# We assume dl_dir (download dir) contains the following
+# directories and files. If not, they will be downloaded
+# by this script automatically.
+#
+#  - $dl_dir/timit
+#      You can find data, train_data.csv, test_data.csv, etc, inside it.
+#      You can download them from https://data.deepai.org/timit.zip
+#
+#  - $dl_dir/lm
+#      This directory contains the language model(LM) downloaded from
+#      https://huggingface.co/luomingshuang/timit_lm, and the LM is based
```

Comment:
Could you please describe how `lm_tgmed.arpa` is obtained?
Is it possible to train it inside icefall?

Reply:
Em....the lm_tgmed.arpa is obtained by this [train_lms.sh](https://github1s.com/luomingshuang/snowfall/blob/timit-recipe/egs/timit/asr/simple_v1/local2/train_lms.sh) which is followed by kaldi. About training lm inside icefall, I think it is a good idea. I have ever considered this problem that if we can train lm with python. There are some methods for it by using [KenLM](https://github.com/kpu/kenlm). Maybe I can have look.

Reply:
ok, `train_lms.sh` uses https://github.com/danpovey/kaldi_lm.git

I will wrap it to Python with pybind11 when I have time.